examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx

```diff
@@ -25,6 +25,7 @@ import {
 import "./BreakoutManager.css";
 import { BreakoutStarter } from "../BreakoutStarter";
 import { WaitingList } from "../WaitingList";
+import { RoomsManager } from "../RoomsManager";
```

Comment:
Imports from design system need to be cleaned up since you now have the RoomsManager component